saleor/graphql/csv/types.py

```diff
@@ -109,8 +110,11 @@ def resolve_app(root: models.ExportFile, info: ResolveInfo):
         return AppByIdLoader(info.context).load(root.app_id) if root.app_id else None
 
     @staticmethod
-    def resolve_events(root: models.ExportFile, _info: ResolveInfo):
-        return root.events.all().order_by("pk")
+    def resolve_events(root: models.ExportFile, info):
+        def _sort_by_pk(records):
```

Comment:
Wouldn't be better to have the ordering applied to the model? It looks like it is missing there.